sdk/openai/Azure.AI.OpenAI/src/Custom/ChatCompletionsOptions.cs

```diff
@@ -83,22 +90,28 @@ public partial class ChatCompletionsOptions
         /// Assistant roles.
         /// </param>
         /// <exception cref="ArgumentNullException"> <paramref name="messages"/> is null. </exception>
-        public ChatCompletionsOptions(IEnumerable<ChatMessage> messages)
+        public ChatCompletionsOptions(IEnumerable<ChatMessage> messages) : this()
```

Comment:
>IEnumerable<ChatMessage> messages [](http://example.com/codeflow?start=38&length=33)

We should also consider adding:
```C#
public ChatCompletionOptions(params ChatMessage[] messages) : this((IEnumerable<ChatMessage>)message)
{}
```

Reply:
I like that idea and I don't think it's problematic for us to add overloads like this -- let's defer the holistic discussion for after this PR, though, as we still have the overarching topic of "how do we want to handle the interplay of required and optional parameters relative to constructor parameters and init-style setters."